flexibleengine/resource_flexibleengine_cce_node_pool_test.go

```diff
@@ -128,50 +128,28 @@ func testAccCheckCCENodePoolExists(n string, cluster string, nodePool *nodepools
 	}
 }
 
-func testAccCCENodePool_Base(rName string) string {
+func testAccCCENodePool_basic(rName string) string {
 	return fmt.Sprintf(`
-data "flexibleengine_vpc_v1" "test" {
-  name = "vpc-default"
-}
-
-data "flexibleengine_vpc_subnet_v1" "test" {
-	name = "subnet-default"
-  }
-
-data "flexibleengine_compute_availability_zones_v2" "test" {}
-
-resource "flexibleengine_compute_keypair_v2" "test" {
-  name = "%s"
-  public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDAjpC1hwiOCCmKEWxJ4qzTTsJbKzndLo1BCz5PcwtUnflmU+gHJtWMZKpuEGVi29h0A/+ydKek1O18k10Ff+4tyFjiHDQAT9+OfgWf7+b1yK+qDip3X1C0UPMbwHlTfSGWLGZquwhvEFx9k3h/M+VtMvwR1lJ9LUyTAImnNjWG7TAIPmui30HvM2UiFEmqkr4ijq45MyX2+fLIePLRIFuu1p4whjHAQYufqyno3BS48icQb4p6iVEZPo4AE2o9oIyQvj2mx4dk5Y8CgSETOZTYDOR3rU2fZTRDRgPJDH9FWvQjF5tA0p3d9CoWWd2s6GKKbfoUIi8R/Db1BSPJwkqB jrp-hp-pc"
-}
-
```

Comment:
please keep data.flexibleengine_compute_availability_zones_v2 and flexibleengine_compute_keypair_v2 as is, then developer can do acc-test with less environment variables.